GVFS/GVFS.Common/GVFSConstants.cs

```diff
@@ -115,6 +115,7 @@ public static class Databases
                 public static readonly string PlaceholderList = Path.Combine(Name, "PlaceholderList.dat");
                 public static readonly string ModifiedPaths = Path.Combine(Name, "ModifiedPaths.dat");
                 public static readonly string RepoMetadata = Path.Combine(Name, "RepoMetadata.dat");
+                public static readonly string GVFSDatabase = Path.Combine(Name, "gvfs.sqlite");

```

Comment:
I don't think we've had much guidance on this, but do you think the filename should use the new name (e.g. "VFSForGit.sqlite")?

I think it's okay to keep the code using the GVFS name for now.

cc: @jrbriggs for his thoughts

Reply:
Given that this is an new file and has no compat concerns I think we switch to the new name to avoid building debt.

Reply:
So is this only for the name of this file or is it for all uses of `GVFS`?  Like should this variable be `VFSForGitDatabase` instead of `GVFSDatabase` and the class `GVFSDatabase`?

Reply:
> So is this only for the name of this file or is it for all uses of GVFS? 

I think the important part is the name of the file.  

 I don't think there'd be any objections to using `VFSForGitDatabase` but I also don't think you have to update the names in code right now either ("GVFS" is still used just about everywhere ).